README.md

```
@@ -65,6 +65,20 @@ curl --socks5-hostname 127.0.0.1:9150 https://ipinfo.tw/ip
 docker stop tor-socks-proxy
 ```
 
+### Using bridges
+
+Edit `/etc/tor/torrc` and the following:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)